week-4/learning-outcomes.md

```diff
@@ -0,0 +1,50 @@
+# Week 4 Learning Outcomes
+
+## Node
+
+### Servers
+
+- [ ] **We can create a web server that responds to http requests**
+- [ ] We can conditionally set ports for our server based on the runtime environment
+- [ ] We can read secrets from a .env file
+- [ ] We can prevent sensitive data from being pushed to GitHub
+
+### GET requests
+
+- [ ] We can route requests to the correct handler function(s)
```

Comment:
This isn't specific to GET requests, should this be moved up to the general "Servers" section?

Reply:
Oli pointed this out too, I've moved it into the 'Servers' section 👍